source/beamline/AMPVControl.cpp

```diff
@@ -89,25 +92,40 @@ void AMReadOnlyPVControl::onReadPVError(int errorCode) {
 	}
 }
 
-
-void AMReadOnlyPVControl::onLowLimitPVValueChanged(double newLowLimit)
+void AMReadOnlyPVControl::setLowLimitValue(double newLowLimit)
 {
-	lowLimitValue_ = newLowLimit;
-	emit minimumValueChanged(lowLimitValue_);
+	if (lowLimitValue_ != newLowLimit) {
```

Comment:
Originally that's the tack I'd taken. I thought that if ahigher level class wants to invite it, they can but they'll still receive the signal.